frontend/styles/theme.js

```diff
@@ -0,0 +1,20 @@
+import { css } from "styled-components";
+
+const colors = {
```

Comment:
why not use the colors from `colors.js`?

Reply:
I can't get that to work. I put ```import colors from "./colors";``` at the top of the docucment but it doesn't recognize the colour variables and throws an error